.github/workflows/go.yml

```
@@ -10,14 +10,14 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.18.x, 1.19.x, 1.20.x]
+        go-version: [1.20.x, 1.21.x]
```

```suggestion
        go-version: [1.19.x, 1.20.x, 1.21.x]
```

(I tend to have one more version than the official Go version)

---

Sure, let me just adapt that.

---

Done, thanks!